### DISCUSSION TITLE
Is Sverchok still in development?

### DISCUSSION THREAD

**Discussion by johntedharvey** [nortikin/sverchok#5139] [General]
Is Sverchok still in development?

**Comment by vicdoval** [comment]
The last commit was 23 days ago so I would say it is

**Comment by johntedharvey** [comment]
Thanks for your email.
I've been trying to find out if there are any plans to fix the viewer draw
node.
Regards
John Harvey

**Reply by gotjoshua** [reply]
Seems this was solved in this issue:

https://github.com/nortikin/sverchok/issues/5125#issuecomment-2351579335
